From 45b3920e21a31b757330e87e6921088515c8a416 Mon Sep 17 00:00:00 2001
From: William <William@undefined.re>
Date: Fri, 12 May 2017 13:57:58 +0200
Subject: [PATCH] ambiguous

---
 src/node_crypto.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/node_crypto.cc b/src/node_crypto.cc
index f18117a41c..061466c31c 100644
--- a/src/node_crypto.cc
+++ b/src/node_crypto.cc
@@ -694,7 +694,7 @@ static int X509_STORE_up_ref(X509_STORE* store) {
 }
 
 static int X509_up_ref(X509* cert) {
-  CRYPTO_add(&cert->references, 1, CRYPTO_LOCK_X509);
+  CRYPTO_add(&(cert)->references, 1, CRYPTO_LOCK_X509);
   return 1;
 }
 #endif  // (OPENSSL_VERSION_NUMBER < 0x10100000L && !OPENSSL_IS_BORINGSSL) || defined(LIBRESSL_VERSION_NUMBER)
-- 
2.12.2

