From 2eb62cc30cc86a86579245c7b00a354a55888673 Mon Sep 17 00:00:00 2001
From: William <wjc@umanlife.com>
Date: Mon, 12 Nov 2018 17:23:44 +0100
Subject: [PATCH] remove default cflags

---
 CMakeLists.txt      | 9 ---------
 test/CMakeLists.txt | 3 ---
 2 files changed, 12 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 01f3122..5303739 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -290,15 +290,6 @@ IF (MSGPACK_BUILD_TESTS)
     ADD_SUBDIRECTORY (test)
 ENDIF ()
 
-IF ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    IF (MSGPACK_ENABLE_SHARED)
-        SET_PROPERTY (TARGET msgpackc APPEND_STRING PROPERTY COMPILE_FLAGS " -Wall -Wextra -DPIC")
-    ENDIF ()
-    IF (MSGPACK_ENABLE_STATIC)
-       SET_PROPERTY (TARGET msgpackc-static APPEND_STRING PROPERTY COMPILE_FLAGS " -Wall -Wextra" )
-    ENDIF ()
-ENDIF ()
-
 IF ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
     IF (MSGPACK_ENABLE_SHARED)
         SET_PROPERTY (TARGET msgpackc APPEND_STRING PROPERTY COMPILE_FLAGS " -Wno-mismatched-tags")
diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 797d700..3379886 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -88,9 +88,6 @@ FOREACH (source_file ${check_PROGRAMS})
         ${CMAKE_THREAD_LIBS_INIT}
     )
     ADD_TEST (${source_file_we} ${source_file_we})
-    IF ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-        SET_PROPERTY (TARGET ${source_file_we} APPEND_STRING PROPERTY COMPILE_FLAGS " -Wall -Wextra")
-    ENDIF ()
     IF ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
         SET_PROPERTY (TARGET ${source_file_we} APPEND_STRING PROPERTY COMPILE_FLAGS " -Wno-mismatched-tags")
     ENDIF ()
-- 
2.19.1

