From 241916e45d43b44833d39f841d68ecb825a38789 Mon Sep 17 00:00:00 2001
From: William <William@undefined.re>
Date: Tue, 25 Apr 2017 12:07:42 +0200
Subject: [PATCH] patch ssl method

---
 src/idevice.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/idevice.c b/src/idevice.c
index ce27495..8f03094 100644
--- a/src/idevice.c
+++ b/src/idevice.c
@@ -678,7 +678,7 @@ LIBIMOBILEDEVICE_API idevice_error_t idevice_connection_enable_ssl(idevice_conne
 	}
 	BIO_set_fd(ssl_bio, (int)(long)connection->data, BIO_NOCLOSE);
 
-	SSL_CTX *ssl_ctx = SSL_CTX_new(SSLv3_method());
+	SSL_CTX *ssl_ctx = SSL_CTX_new(SSLv23_method());
 	if (ssl_ctx == NULL) {
 		debug_info("ERROR: Could not create SSL context.");
 		BIO_free(ssl_bio);
-- 
2.12.2

